packages/ibm-products/src/components/APIKeyModal/APIKeyDownloader.js

```diff
@@ -44,6 +45,8 @@ export const APIKeyDownloader = ({
         <a
           {...linkProps}
           className={`${pkg.prefix}--apikey-modal__download-link`}
+          aria-label={DownloadLinkLabel}
```

Comment:
Add `aria-label={DownloadLinkLabel?? linkText}` since `DownloadLinkLabel` is optional